EndToEndSetupGuide.md

```diff
@@ -62,157 +81,137 @@ git clone https://github.com/mcode/rems-setup.git
-
-	`cd test-ehr`
-
-- Run
-
-	`gradle bootRun`
-
-- Load Data (in separate window, also in repo folder)
-
-	`gradle loadData`
+	```bash
+	cd test-ehr
+	
+	# Run
+	gradle bootRun
+	```
+	
+	```bash
+	# Load Data (in separate window, also in repo folder)
+	gradle loadData
+	```
 
 ### request-generator
 
 - Navigate into directory already cloned from GitHub [www.github.com/mcode/request-generator](https://www.github.com/mcode/request-generator)
+	```bash
+	cd request-generator
 
-	`cd request-generator`
-
-- Setup
-
-	`npm install`
-
-- Run
+	# Setup
+	npm install
 
-	`npm start`
+	# Run
+	npm start
+	```
 
 ## Core Applications
 
 ### rems-admin
 - Navigate into directory already cloned from GitHub [www.github.com/mcode/rems-admin](https://www.github.com/mcode/rems-admin)
-
-	`cd rems-admin`
-
-- Submodule Initialization
-
-	`git submodule update --init`
-	
 - Update env.json
 	- Add your VSAC key to env.json as the default value for `VSAC_API_KEY`
-
-- Setup
-
+	```bash
+	cd rems-admin
+	
+	# Submodule Initialization
+	git submodule update --init
+	
+	# Setup
 	`npm install`
 
-- Run
-
-	`npm start`
+	# Run
+	npm start
+	```
 
 ### pims
 - Navigate into directory already cloned from GitHub [www.github.com/mcode/pims](https://www.github.com/mcode/pims)
+	```bash 
+	cd pims
+	```
+	
+	Backend
+	```bash 
+	# Navigate to the backend directory
+	cd backend
 
-	`cd pims`
-
-- Backend
-	- Navigate to the backend directory
-		
-		`cd backend`
-	- Setup
-
-		`npm install`
-	- Run
+	# Setup
+	npm install
 
-		`npm start`
+	#Run
+	npm start
+	```
 		
-- Frontend
-	- Navigate to the frontend directory
-
-		`cd frontend`
-	- Setup
-
-		`npm install`
-	- Run
-		- Linux or Mac
-
-			`npm start`
-		- Windows
-
-			`PORT=5050 npm start`
- 
-### dtr
-- Navigate into directory already cloned from GitHub [www.github.com/mcode/dtr](https://www.github.com/mcode/dtr)
-
-	`cd dtr`
-
-- Setup
-
-	`npm install`
-
-- Run
+	Frontend
+	```bash 
+	# Navigate to the frontend directory
+	cd frontend
+	
+	# Setup
+	npm install
 
-	`npm start`
+	# Run
+	npm start # Mac or Linux
+	PORT=5050 npm start # Windows
+	```
 
 ### rems-smart-on-fhir
-- Navigate into directory already cloned from GitHub [www.github.com/mcode/rems-smart-on-fhir](https://www.github.com/mcode/rems-smart-on-fhir)
-
-	`cd rems-smart-on-fhir`
+- Navigate into directory already cloned from GitHub [https://www.github.com/mcode/rems-smart-on-fhir](https://www.github.com/mcode/rems-smart-on-fhir)
+	```bash 
+	# Navigate into directory already cloned from GitHub
+	cd rems-smart-on-fhir
 
-- Submodule Initialization
+	# Submodule Initialization
+	git submodule update --init
 
-	`git submodule update --init`
+	# Setup
+	npm install
 
-- Setup
-
-	`npm install`
-
-- Run
-	- Linux or Mac
-
-		`npm start`
-	- Windows
-
-		`PORT=4040 npm run start`
-	
-	
+	# Run
+	npm start # Mac or Linux
+	PORT=4040 npm run start # Windows
```

Comment:
do we need port here for windows?

Reply:
Yes, these are on purpose.

Reply:
The port was only needed on windows, but you could just specify it for all. This seems weird to put the comment # Windows on the same line, seems wrong to me.

Reply:
Having a comment on the line with the code seems like a bad idea when someone is trying to copy and paste it in a terminal.